language/vm/src/errors.rs

```diff
@@ -2,23 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::IndexKind;
-use libra_types::{
+use move_core_types::{
     account_address::AccountAddress,
-    transaction::TransactionStatus,
     vm_error::{StatusCode, VMStatus},
 };
 
-// We may want to eventually move this into the VM runtime since it is a semantic decision that
```

Comment:
do we need the prologue errors here, they hardly are a VM thing.... 
it would be good to remove this file entirely

Reply:
Good point. I think these should be moved. I'll move these to `vm_errors` if that sounds good to you?

Reply:
I think it's more what it dons good to you ;)
this file for me was always odd and it never had anything VM strictly speaking. `Location` will be removed soon and you can stash it anywhere.
Another PR fine too to me. 
Thanks!

Reply:
I'll put it up in another PR and scatter the functions and whatnot around appropriately :)